content/en/docs/_index.md

```diff
@@ -29,5 +29,58 @@ Operators must claim ownership of their APIs. OLM will prevent conflicting Opera
 ### Declarative UI controls
 Operators can behave like managed service providers. Their user interface on the command line are APIs. For graphical consoles OLM annotates those APIs with descriptors that drive the creation of rich interfaces and forms for users to interact with the Operator in a natural, cloud-like way. 
 
+
+## How OLM solves the cluster admin and developer needs.
+
+### Operator Installation and Management
+
+This packaging mechanism helps to simplify the multiple steps involved in deploying an Operator. OLM fulfills necessary metadata for visualizing them in compatible UIs, including installation instructions and API hints in the form of CRD descriptors. It helps cluster admin, developers to manage the lifecycle of all Operators and their associated services running across their clusters.
+
+### Dependency Resolution
+
+Using OLM eliminates the need for developers and cluster admins to have to worry about managing dependancies when developing one or more operators. OLM offers dependency resolution and upgrades the lifecycle of running operators. This functionality helps operator developers to resolve the problems related to dependency management.
+
+### Works similar to package Managers
+
+Operator developers may need to use one or more APIs for the operator installation and OLM helps solve this use case by verifying that all of the required APIs are available for the operator. If they are not available, then OLM will not install the operator.
+
+### Operator Groups
+
+To give cluster admins complete control over the permissions that OLM grants operators, OLM introduced the concept of `OperatorGroups`.
+
+* Defines the set of permissions that OLM may grant to member operators
+* Defines the set of namespaces that OLM may grant namespaced permissions in.
+
+## Building blocks of OLM
+
+### [ClusterServiceVersion](/docs/concepts/crds/clusterserviceversion)
+A `ClusterServiceVersion (CSV)` represents a particular version a running operator on a cluster. It includes metadata such as name, description, version, repository link, labels, icon, etc. 
+
+
+### [CatalogSource](/docs/concepts/crds/catalogsource/)
+A `CatalogSource` represents a store of metadata that OLM can query to discover and install operators and their dependencies.
+
+
+### [OperatorCondition](/docs/concepts/crds/operatorcondition/)
+An `OperatorCondition` is CustomResourceDefinition that creates a communication between OLM and an operator it manages. Operators may write to the `Status.Conditions` array to modify OLM management the operator.
```

Comment:
Please change the text to read as follows:
An `OperatorCondition` is a CustomResourceDefinition that creates a communication between OLM and an operator it manages. Operators may write to the `Status.Conditions` array to modify the OLM operator management.

Reply:
Modified in #135